csrc/device_lower/utils.h

```diff
@@ -185,71 +185,6 @@ std::vector<Expr*> replaceInputsInExpr(
     const std::vector<Expr*>& exprs,
     const std::unordered_map<Val*, Val*>& replacement_map);
 
-// Go through all expressions and compute a local ordering of loops. operator<
```

Comment:
Just moved to the bottom of this file but to the namespace of `lower_utils` as this is only for lowering